packages/atomic/src/pages/examples/commerce-website/commerce-nav.mjs

```diff
@@ -9,7 +9,7 @@ export const navContent = {
   'Surf accessories': {
     href: 'listing-surf-accessories.html',
     label: 'Surf accessories',
-    barcaUrl: `${barcaRoot}/browse/promotions/surf-with-us-this-year`,
+    barcaUrl: `${barcaRoot}/browse/promotions/accessories/surf-with-us-this-year`,
```

Comment:
?

Reply:
The url changed in Barca, Oli gave me this new one :)